### PR TITLE
Add name_pending RPC command.

### PR DESCRIPTION
**Pull request by domob1812**
Implement `name_pending` based on the RPC command of Huntercoin / the old client.  See also the discussion in https://github.com/namecoin/namecore/issues/24.

### CODE REVIEW

File: src/rpcnames.cpp [hunk 1]
```diff
@@ -435,6 +440,106 @@ name_filter (const UniValue& params, bool fHelp)
...
+        "    \"op\": xxxx       (string) the operation being performed\n"
+        "    \"name\": xxxx     (string) the name operated on\n"
+        "    \"value\": xxxx    (string) the name's new value\n"
+        "    \"txid\": xxxx     (string) the txid corresponding to the operation\n"
+        "    \"ismine\": xxxx   (boolean) whether the name is owned by the wallet\n"
+        "  },\n"
+        "  ...\n"
+        "]\n"
+        + HelpExampleCli ("name_pending", "")
+        + HelpExampleCli ("name_pending", "\"d/wikileaks\"")
```
Comment (JeremyRand) [inline src/rpcnames.cpp]: Minor nitpick: d/wikileaks is, to our knowledge, not controlled by WikiLeaks, therefore we probably shouldn't use it as an example.

Comment (domob1812) [inline src/rpcnames.cpp]: Feel free to suggest another example name. :)

**Comment by phelixbtc** [comment]
ACK. This should be quite helpful with nameGUI.

File: src/rpcnames.cpp (hunk 1 above)
Comment (JeremyRand) [inline src/rpcnames.cpp]: How about d/domob ?  :)

**Comment by domob1812** [comment]
I changed the test name to "d/domob" as per @JeremyRand's suggestion.  Closing this PR now, I will push a squashed commit to master.

**Pull request closed without merging**